### PR TITLE
Fix annual report broken links

### PR DESCRIPTION
**Pull request by MadhavJivrajani**
Fix broken link for api machinery's annual report.

Signed-off-by: Madhav Jivrajani <madhav.jiv@gmail.com>

### CODE REVIEW

**Comment by MadhavJivrajani** [comment]
The following don't have annual reports to be linked:
- SIG Service Catalog
- SIG Testing
- WG IoT Edge
- WG Component Standard

**Comment by mrbobbytables** [comment]
huh - thought we got all of them.

SIG Service Catalog and SIG Testing have reports that will merge eventually.

The others don't have any.

Will fix them in a follow up

File: reports/summary-2020.md [hunk 1]
```diff
@@ -696,7 +696,7 @@ initiative with supporting links to KEPs and more.
     - Kubefed - support for pull-based reconciliation 
     - Work API - road to alpha
 
-- [SIG Network](https://git.k8s.io/community/sig-network/annual-report-2020.md#current-initiatives-and-project-health)
+- [SIG Network](https://git.k8s.io/community/sig-network/annual-report-2021.md#current-initiatives-and-project-health)
```
Comment (mrbobbytables) [inline reports/summary-2020.md:699]:
```suggestion
- [SIG Network](https://git.k8s.io/community/sig-network/annual-report-2020.md#current-initiatives-and-project-health)
```

This should be fixed in k/community

Comment (MadhavJivrajani) [reply reports/summary-2020.md]: Sounds good, I'll create an issue there 👍

**Comment by MadhavJivrajani** [comment]
Issue created in k/community for sig-network's annual report link: https://github.com/kubernetes/community/issues/5838

**Comment by MadhavJivrajani** [comment]
> The others don't have any.

Should I remove hyperlinks to these ones?

**Comment by mrbobbytables** [comment]
@MadhavJivrajani that would be great 🙏 I was planning on following it up in another PR, but if you want to roll it into that one it saves a step 👍

**Comment by MadhavJivrajani** [comment]
done 👍🏻 
thanks :)

**Review by mrbobbytables — COMMENTED** [review]
/lgtm
/approve

Thanks!

**Pull request merged**